spark/src/main/scala/com/alibaba/graphar/EdgeInfo.scala

```diff
@@ -426,6 +426,10 @@ class EdgeInfo() {
     str = prefix + getAdjListPrefix(adj_list_type) + str
     return str
   }
+
+  def getConcatKey(): String = {
+    return getSrc_label + GeneralParams.regularSeperator + getEdge_label + GeneralParams.regularSeperator + getDst_label
```

Comment:
We could modify the GraphTransformer(line 108) to use this function

Reply:
Fixed.